its/core-it-support/core-it-plugins/maven-it-plugin-active-collection/src/main/java/org/apache/maven/plugin/coreit/CheckThreadSafetyMojo.java

```diff
@@ -95,7 +95,11 @@ public void run() {
                     getLog().info("[MAVEN-CORE-IT-LOG] Thread " + this + " uses " + tccl);
                     Thread.currentThread().setContextClassLoader(tccl);
                     while (go.isEmpty()) {
-                        // wait for start
+                        try {
+                            Thread.sleep(100); // wait for the start
+                        } catch (InterruptedException e) {
```

Comment:
No, don't throw a raw RuntimeException. probably just continue here

Reply:
ide finding: `'continue' is unnecessary as the last statement in a loop `
<img width="1156" alt="image" src="https://github.com/user-attachments/assets/6d50f67a-58f7-4ce2-88f9-887ba6b37c80" />